l10n_it_reverse_charge/README.rst

```diff
@@ -119,7 +119,8 @@ Contributors
 
 * Davide Corio
 * Alex Comba <alex.comba@agilebg.com>
-* Lorenzo Battistini <lorenzo.battistini@agilebg.com
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
```

Comment:
You can autogenerate a much nicer README using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)